topiary/src/atom_collection.rs

```diff
@@ -67,6 +67,26 @@ pub struct AtomCollection {
 }
 
 impl AtomCollection {
+    /// Returns a basic AtomCollection with the supplied atoms. Only used for
+    /// testing. Normally you should use `AtomCollection::collect_leafs`
+    /// instead.
+    pub fn new(atoms: Vec<Atom>) -> Self {
```

Comment:
Unless we find a use outside of tests for this one, I think it would be nice to use `#[cfg(test)]` here.

Reply:
Fixed.